docs/index.rst

```diff
@@ -30,7 +30,10 @@
    :caption: Messaging Protocols
    :hidden:
 
-   intro
+   communication_protocols/iquart_protocol
```

Comment:
The IQUART messaging stuff is no longer called "intro", now it lives with all of the communication protocols in this new directory